README.md

```diff
@@ -201,6 +201,16 @@ The files uploaded to your Active Storage service provider will be renamed to
 include an ISO 8601 timestamp and the Task name in snake case format. The CSV is
 expected to have a trailing newline at the end of the file.
 
+The `#count` load and parse the entire file to know the accurate number of rows.
+With files with millions of rows, it take several seconds to process. You might
+consider to skip the count or use an approximative approach, eg: count the
+number of new lines:
```

Comment:
We need to remove the bit about the newline at the end of file since it won't be necessary anymore.

```suggestion
include an ISO 8601 timestamp and the Task name in snake case format.

The implicit `#count` method loads and parses the entire file to determine the
accurate number of rows. With files with millions of rows, it takes several
seconds to process. Consider skipping the count (defining a `count` that returns
`nil`) or use an approximation, eg: count the number of new lines:

```

Also reworded a bit.